src/Dhadgar.Cli/CLAUDE.md

```
@@ -3,21 +3,52 @@
+
 Automatically installs as `dhadgar` global tool during build:
+
 ```bash
 dotnet build src/Dhadgar.Cli -c Release
 dhadgar --help
 ```
 
+## Command Groups
+
+| Command | Description |
+|---------|-------------|
```

markdownlint-cli2 (0.20.0)

Table column style
Table pipe is missing space to the right for style "compact"

(MD060, table-column-style)